src/cpp/include/openvino/genai/whisper_generation_config.hpp

```diff
@@ -97,6 +97,8 @@ class OPENVINO_GENAI_EXPORTS WhisperGenerationConfig : public GenerationConfig {
     // A list containing the non-speech tokens that will be suppressed during generation.
     std::vector<int64_t> suppress_tokens;
 
+    bool apply_chat_template = false;
```

Comment:
I suspect that shadows the original variable instead of overriding its value. The solution would be to set the value in WhisperGenerationConfig's constructor

Reply:
moved to constructor